lib/commands/find.js

```diff
@@ -3,25 +3,29 @@ let commands = {}, helpers = {}, extensions = {};
 // Override the following function for your own driver, and the rest is taken
 // care of!
 
-// stategy: locator strategy
+//helpers.findElOrEls = async function (strategy, selector, mult, context) {}
+// strategy: locator strategy
 // selector: the actual selector for finding an element
 // mult: multiple elements or just one?
 // context: finding an element from the root context? or starting from another element
-//helpers.findElOrEls = async function (strategy, selector, mult, context) {}
 
 commands.findElement = async function (strategy, selector) {
+  this.validateLocatorStrategy(strategy, !!this.context);
```

Comment:
removed it, overriding validateLocatorStrategy in ios driver, still it feels strange, the webcontext code could be more consistent.